src/core/tsi/ssl_transport_security_utils.cc

```diff
@@ -313,4 +313,44 @@ absl::StatusOr<std::string> IssuerFromCert(X509* cert) {
   return ret;
 }
 
+absl::StatusOr<std::string> AkidFromCertificate(X509* cert) {
```

Comment:
Please check that `cert` is not null, otherwise we will segfault when we pass it into `X509_get_ext_by_NID`.

Reply:
Done